backend/tests/routes/broadcast/getBroadcasts.test.js

```diff
@@ -10,15 +10,14 @@ describe('Test for get Broadcasts:', () => {
   it('get broadcasts from DB at /broadcast', (done) => {
     const query = {
       page: 2,
-      limit: 2,
+      tags: 'tag1,tag2',
```

Comment:
is this getting stored as array? check once

Reply:
firstly using these tags all the broadcasts are fetched from DB and then they are sorted on the basis of their Created Date in decreasing order(which means the most recent post from the broadcasts will be first and so on).